web/src/stories/Breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react'
+import { ComponentStory, ComponentMeta } from '@storybook/react'
+import '../variables.scss'
+
+import Breadcrumbs from '../components/Breadcrumbs/Breadcrumbs'
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'Expanded View/Breadcrumbs',
+  component: Breadcrumbs,
+  // More on argTypes: https://storybook.js.org/docs/react/api/argtypes
+  argTypes: {
+    backgroundColor: { control: 'color' },
+  },
```

Comment:
```suggestion
  // More on argTypes: https://storybook.js.org/docs/react/api/argtypes
  // argTypes: {
  //  backgroundColor: { control: 'color' },
  // },
```
I think we should do a sweep and comment these out, or otherwise figure out what we want to use these special Arg types for and what options are available, and make good use of them.